students/marsha_w/lesson02/grid_printer_exercise.py

```diff
@@ -0,0 +1,21 @@
+'''
+Title:Grid Printer Exercise
+Dev:Marsha Wheeler
+Date:01-23-18
+'''
+
+#Write a function that draws a Grid and uses two parameters in the function
+def drawGrid(nRows, nCells):
```

Comment:
python uses `snake_case` for all function and variable names.  Only use `CamelCase` for class names.